storage/tests/unit/test_bucket.py

```diff
@@ -710,6 +710,43 @@ def test_labels_setter(self):
         self.assertIsNot(bucket._properties['labels'], LABELS)
         self.assertIn('labels', bucket._changes)
 
+        # Make sure that a update call correctly adds the labels.
+        client = mock.NonCallableMock(spec=('_connection',))
+        client._connection = mock.NonCallableMock(spec=('api_request',))
+        bucket.update(client=client)
+        client._connection.api_request.assert_called()
+        _, _, kwargs = client._connection.api_request.mock_calls[0]
+        self.assertEqual(len(kwargs['data']['labels']), 2)
+        self.assertEqual(kwargs['data']['labels']['color'], 'red')
+        self.assertEqual(kwargs['data']['labels']['flavor'], 'cherry')
```

Comment:
The extra call to `update` feels out of place for this test:  unit test testcase methods should limit scope to the thing-under-test.